packages/server/src/automations/steps/updateRow.ts

```diff
@@ -35,6 +35,11 @@ export const definition: AutomationStepSchema = {
           customType: AutomationCustomIOType.ROW,
           title: "Table",
         },
+        oldRow: {
+          type: AutomationIOType.OBJECT,
+          customType: AutomationCustomIOType.ROW,
+          title: "Row",
```

Comment:
should this have a title of "Row" - should it not be "Old Row"?

Reply:
I honestly wasn't sure about this lol. Where is the title even used - I had assumed the title was to do with the type of the binding (i.e a row) for some reason. I can change it for sure though